Splash/Splash/Routine/RoutineCell.m

```diff
@@ -0,0 +1,12 @@
+//
+//  RoutineCell.m
+//  Splash
+//
+//  Created by Shreya Vinjamuri on 8/2/22.
+//
+
+#import "RoutineCell.h"
+
+@implementation RoutineCell
```

Comment:
Do you plan to add more implementation here as follow up?

Reply:
Yes, I have.